packages/twenty-front/src/modules/workspace-invitation/graphql/mutations/removeInvitation.ts

```diff
@@ -0,0 +1,7 @@
+import { gql } from '@apollo/client';
+
+export const DELETE_INVITATION = gql`
+  mutation DeleteCustomInvitation($invitationId: String!) {
```

Comment:
Naming could be better but I lost a lot of time when I didn't understand why "DeleteInvitation" did not work. 

I am open to suggestions.

Reply:
Why didn't DeleteInvitation work? 
Could be DeleteWorkspaceInvitation eventually, custom is not a neutral term in our codebase, it usually means there's a "standard" alternative that goes in pair with the custom (it isn't the case here)